src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -365,6 +371,36 @@ public override Task SendUsersAsync(IReadOnlyList<string> userIds, string method
             await ack;
         }
 
+        private async Task RemoveUserAsync(HubConnectionContext connection)
+        {
+            var userChannel = _channels.User(connection.UserIdentifier);
```

Comment:
Why is this called at the top of the method? Should the call be moved to where it is used?